optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -596,21 +626,29 @@ def forward(
             encoder_outputs = self.encoder(input_ids=input_ids, attention_mask=attention_mask)
 
         # Decode
+        decoder_inputs = {
+            "encoder_hidden_states": encoder_outputs.last_hidden_state,
+            "encoder_attention_mask": attention_mask,
+        }
```

Comment:
I would be in favor of removing the `decoder_inputs` dictionnary and just add the labels arguments to both the `decoder` / `decoder_with_past`

Reply:
I get what you mean, will put the original inference back.